sqlalchemy_redshift/dialect.py

```diff
@@ -676,11 +677,9 @@ def _get_table_or_view_names(self, relkind, connection, schema=None, **kw):
     def _get_column_info(self, *args, **kwargs):
         kw = kwargs.copy()
         encode = kw.pop('encode', None)
-        if sa.__version__ >= '1.2.0':
-            # SQLAlchemy 1.2.0 introduced a required 'comment' param
-            kw['comment'] = kw.get('comment', None)
-        else:
-            kw.pop('comment', None)
+        if sa.__version__ < '1.2.0':
+            # SQLAlchemy 1.2.0 introduced the 'comment' param
+            del kw['comment']
```

Comment:
Can you explain why we need to invert this condition here? Is there ever a possibility that 'comment' will not exist in kwargs, and this could raise an exception?

Reply:
Nope, the only place this method is called is on line ~480 (above), and I've added the `comment` kwarg there, so this will never raise a `KeyError`.

It makes sense to invert the condition because otherwise it would be something like

```python
if sa.__version__ >= '1.2.0':
    pass
else:
    del kw['comment']
```